tests/framework/context/test_context.py

```diff
@@ -189,8 +195,13 @@ def extra_params(request):
 
 @pytest.fixture
 def dummy_context(tmp_path, prepare_project_dir, env, extra_params):
-    configure_project(MOCK_PACKAGE_NAME)
-    config_loader = OmegaConfigLoader(str(tmp_path / "conf"), env=env)
+    bootstrap_project(tmp_path)
+
+    session = KedroSession.create(
+        project_path=tmp_path, env=env, extra_params=extra_params
+    )
+    context = session.load_context()
+    config_loader = context.config_loader
```

Comment:
The test was never updated after session is created. This is causing fail test because settings will not propagate  correctly if we try to construct KedroContext directly.